packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/Palette.tsx

```diff
@@ -138,36 +145,32 @@ export const Palette = ({
 
   const { hideDiagramPalette } = useDiagramPalette();
   const { hideDiagramElementPalette } = useDiagramElementPalette();
-  const { hideGroupPalette } = useGroupPalette();
```

Comment:
Why is this one removed from here? I'm not sure why it got added in the first place